schema-extended/arxiv-extended.json

```diff
@@ -0,0 +1,707 @@
+            "type": "string",
+            "format": "date-time"
+          },
+          "announced_date": {
+            "title": "Announced date",
+            "description": "Date this version was announced.",
+            "type": "string",
+            "format": "date-time"
+          },
+          "license": {
+            "title": "License",
+            "description": "The license associated with the content.",
+            "oneOf": [
+              {"$ref": "#/definitions/active_licenses"},
+              {"$ref": "#/definitions/inactive_licenses"}
+            ]
+          },
+          "titles": {
+            "title": "Titles",
+            "description": "The English and non-English versions of the title.",
+            "$ref": "#/definitions/titles"
+          },
+          "abstracts": {
+            "$ref": "#/definitions/abstracts"
+          },
+          "legacy_authors": {
+            "title": "Legacy authors",
+            "description": "The legacy authors field, which is an unstructured string.",
+            "type": "string",
+            "minLength": 1,
+            "examples": [
+              "J. Doe (Cornell University), M. Smith (Stanford University), F. Bar"
+            ]
+          },
+          "authors": {
+            "title": "Authors",
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/person"
+            },
+            "minItems": 1
+          },
+          "collaborations": {
+            "title": "Collaborations",
+            "description": "Collaborations associated with this e-print. Should not include 'Collaboration' suffix.",
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/collaboration"
+            },
+            "examples": [
+              "ATLAS",
+              "BABAR",
+              "L3",
+              "ZEUS"
+            ],
+            "minItems": 1,
+            "uniqueItems": true
+          },
+          "comments": {
+            "title": "Comments",
+            "description": "Comments from authors or arXiv administrators.",
+            "type": "object",
+            "properties": {
+              "author_comments": {
+                "Title": "Author comments",
+                "description": "Comments from authors.",
+                "type": "array",
+                "items": {
+                  "$ref": "#/definitions/comment"
+                },
+                "minItems": 1,
+                "uniqueItems": true
+              },
+              "administrator_comments": {
+                "title": "Administrator comments",
+                "description": "Comments from by arXiv administrators",
+                "type": "array",
+                "items": {
+                  "$ref": "#/definitions/comment"
+                },
+                "minItems": 1,
+                "uniqueItems": true
+              },
+              "legacy_comments": {
```

Comment:
Do the current administrator comments follow a pattern? from a quick glance they seem to start with `arXiv admin note: `. If so, it should be possible to split the comments into author comments and administrator comments and drop this field.

Reply:
Yes, for the most part they do follow a pattern, but it's less consistent for older papers.